nyaa/templates/rss.xml

```diff
@@ -40,6 +40,11 @@
 			<nyaa:categoryId>{{- cat_id }}</nyaa:categoryId>
 			<nyaa:category>  {{- category_name(cat_id) }}</nyaa:category>
 			<nyaa:size>      {{- torrent.filesize | filesizeformat(True) }}</nyaa:size>
+			<nyaa:submitter> {{- 'Anonymous' if torrent.anonymous or not torrent.user else torrent.user.username }}</nyaa:submitter>
```

Comment:
`torrent.user` won't work on ES - you'll have to work around it as above with the category id.

Reply:
I checked the code, on Elastic Search you use `torrent.sub_category_id` instead of `torrent.sub_category.id_as_string`. How did you make the `sub_category.id_as_string` value available in `torrent.sub_category_id`?

I'm guessing I need to do something similar with the torrents username. Something like:
```j2
{% set submitter = use_elastic and torrent.submitter or torrent.user.username %}
```

I'm not sure how I can map the username to `torrent.submitter` though

Reply:
We don't make anything look like anything. You can see [just above](https://github.com/nyaadevs/nyaa/pull/481/files/29ccd939f8ac0f906804d57ea41e3607c5795397#diff-0c21456d97f5b069933b89256c6c1573R39) how we choose either the ES category field or the DB model's relationship field. You don't have to map that back to anything, just use the variable you just `set`.

Reply:
I get that but like how's the fields made? Or where can I find the ES field for username?

Reply:
I checked the ES Mappings and I saw you have a `uploader_id` in there. Is that the user's username or his account ID?

Reply:
`torrent` is either `models.Torrent` or an ES object with the fields listed in https://github.com/nyaadevs/nyaa/blob/master/es_mapping.yml#L84 when `use_elastic` is `True`.

Reply:
I already looked at the ES mappings but found no field for username or submitter, what value am I meant to show on ES?

Reply:
Oh shit, you're right, I remembered falsely (since `uploader_id` is there). This will complicate things, since we'll have to decide whether or not to store the uploader name in ES. I'll get back on this later.

Reply:
Any updates? If its too much work then I can remove the "submitter" from the pull requests since its not really needed. If someone needs it then it can be added later in another pull requests.

Reply:
@TheAMM Can you give me update